plugins/modules/swim_workflow_manager.py

```diff
@@ -1703,13 +1715,14 @@ def get_diff_distribution(self):
                         self.result['changed'] = True
                         self.status = "success"
                         self.single_device_distribution = True
-                        self.result['msg'] = "Image with Id {0} Distributed Successfully".format(image_id)
-                        self.result['response'] = self.msg
+                        self.result['msg'] = "Image with Id {0} Distributed Successfully for the device: {1} ".format(image_id, device_ip)
```

Comment:
self.result['msg'] = "Image '{0}' with ID {1} has been successfully distributed to device: {2}".format(image_name, image_id, device_ip)
OR
self.result['msg'] = "Successfully distributed image '{0}' (ID: {1}) to device: {2}".format(image_name, image_id, device_ip)

Reply:
updated the code